sources/packages/web/src/assets/css/formio-shared.scss

```diff
@@ -591,3 +591,49 @@
   font-family: "BCSans-bold", "BCSans", "Noto Sans", Verdana, Arial, sans-serif;
   color: $brand-blue;
 }
+
+.restrictions-details {
+  font-style: $font-style-normal;
+  font-weight: $font-weight-normal;
+  font-size: $font-size-xsmall;
+  line-height: $line-height-header-xsm;
+}
+
+/* Grid for info with icon to the left */
+.info-grid-icon {
```

Comment:
if it is for the color. I will say, we have we can do something like this so that, it will fix all formio info banners, without form definition change
![image](https://user-images.githubusercontent.com/77353155/197048310-2f2822eb-2eb2-4bf0-83c2-45b22db70ab6.png)

Reply:
![image](https://user-images.githubusercontent.com/77353155/197048488-83747810-1052-4834-b656-aeb7e1aacd22.png)

Reply:
That is to define a grid area.

Reply:
As we talked let's follow the way as the other info banner are already implemented and remove the extra styles.

Reply:
It was removed as per our conversation. Thanks.